src/components/Config/Config.tsx

```diff
@@ -120,68 +146,148 @@ function Config() {
     }
   }, [checkConnection, wallet]);
 
+  const ArbitrumNetworkConnect = useCallback(async () => {
+    const ArbitrumTestChainId = '0x66eeb'; // arbitrum test net
+    if (typeof window.ethereum !== 'undefined') {
+      const chainId = await window.ethereum.request({ method: `eth_chainId` });
+      if (chainId === ArbitrumTestChainId) {
+        notification.success({ message: `You're connected to Automate!` });
+      } else {
+        console.log('oulalal, switch to the correct network,chainid: ' + chainId);
+        try {
+          await window.ethereum.request({
+            method: 'wallet_switchEthereumChain',
+            params: [{ chainId: ArbitrumTestChainId }],
+          });
+          console.log('You have succefully switched to Arbitrum Test network');
+        } catch (chainerror) {
+          if (chainerror.code === 4902) {
+            console.log('This network is not available in your metamask, please add it');
+          }
+          try {
+            await window.ethereum.request({
+              method: 'wallet_addEthereumChain',
+              params: [
+                {
+                  chainId: ArbitrumTestChainId,
+                  chainName: 'Arbitrum Testnet', // Automate Draft Arbitrum
+                  rpcUrls: ['https://rinkeby.arbitrum.io/rpc'], // url + draft=true
+                  blockExplorerUrls: ['https://testnet.arbiscan.io/'],
+                  nativeCurrency: {
+                    symbol: 'ETH', // 2-6 characters long
+                    decimals: 18,
+                  },
+                },
+              ],
+            });
+          } catch (addError) {
+            console.log(addError);
+          }
+        }
+      }
+    } else {
+      notification.error({
+        message: (
+          <span>
+            Metamask is not installed. Metamask is required to connect Automate. Install the{' '}
+            <a href="https://metamask.io/download/" target="_blank" rel="noopener noreferrer">
+              Metamask extension.
+            </a>{' '}
+            If you have installed refresh the page.
+          </span>
+        ),
+      });
+    }
+  }, []);
+
   return (
     <Container>
       <PageTitle title="Connect" />
+
       <Typography.Title level={3} className="title">
-        {completed ? 'Congratulations!' : 'Connection Settings'}
+        {completed ? 'Congratulations!' : networkvalue < 2 ? 'Connection Settings' : 'Select Network'}
```

Comment:
Magic value `2` used here. Also it's very confusing that you're using `<` instead of `===` since network ids are very specific and it doesn't make sense here to 'relax' the check to `<`. Use the network id from the `enum` mentioned earlier and do a strict check